common/authorization/authorizer.go

```diff
@@ -87,6 +87,8 @@ func GetAuthorizerFromConfig(config *config.Authorization) (Authorizer, error) {
 		return NewNoopAuthorizer(), nil
 	case "default":
 		return NewDefaultAuthorizer(), nil
+	case "opa":
```

Comment:
There's nothing really specific to OPA at all here, is there? Should we call it a `http` authorizer?

Reply:
Yep HTTP makes more sense. I will update that.